google/auth/credentials.py

```diff
@@ -277,7 +286,7 @@ class Scoped(ReadOnlyScoped):
     """
 
     @abc.abstractmethod
-    def with_scopes(self, scopes):
+    def with_scopes(self, scopes, default_scopes=None):
```

Comment:
Is adding an additional optional parameter to an abstract method a breaking change?

Reply:
It seems fine. 
https://stackoverflow.com/questions/42778784/abstract-classes-with-varying-amounts-of-parameters